src/table/TableWrapper.jsx

```diff
@@ -66,7 +66,7 @@ export default function TableWrapper(props) {
   useEffect(() => {
     const updateSize = () => setTableWidth(rootElement.clientWidth);
     window.addEventListener('resize', updateSize);
-    tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));
+    tableSection.current?.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));
```

Comment:
this one sometimes is unreliable, if it was based on typescript, it would still give you some errors here. 
i think it's better to specifically check for this one in a `if` statement in the beginning of the `useEffect`. like 
```
if(!tableSection.current) return;
```

Reply:
sure. Changed to 
`tableSection.current && tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));`

Worth to mention that it is only a workaround for the unit testing issue. in unit testing, tableSection.current is regarded as null.

Reply:
Great, it's looking good to me ✅